.github/workflows/bot-pr-comment.yml

```diff
@@ -46,8 +35,9 @@ jobs:
         persist-credentials: false
     - name: Commit formatting changes
       env:
-        GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
What's the reason behind this change?

Reply:
To get the CI running in my fork. I still have to remove some commits before I would merge this,

Reply:
To get the CI running on my fork. I will remove this and other commits before merging.